### PR TITLE
Renaming the term GS

### PR DESCRIPTION
**Pull request by sushmangupta**

### CODE REVIEW

File: products/guided-shopping/README.md [hunk 1]
```diff
@@ -1,18 +1,18 @@
...
-Guided Shopping is a commercial extension and is not available as open source.
+DSR is a commercial extension and is not available as open source.
 {% endhint %}
 
 ## Prerequisites
 
-Review the below minimum operating requirements before you install the Guided Shopping feature:
+Review the below minimum operating requirements before you install the DSR feature:
 
 * Instance of [Shopware 6](../../guides/installation/legacy/from-scratch.md) (version 6.4.18.0 and above).
```
Comment (sushmangupta) [inline products/guided-shopping/README.md:17]: This is not a recommended type of installation. Do you want to suggest any other ?

File: products/guided-shopping/checklist.md [hunk 2]
```diff
@@ -19,7 +19,7 @@
 
 ### Plugin Setup
 
-![ ](../../.gitbook/assets/products-guidedShopping-checklist.png)
+![ ](../../.gitbook/assets/products-digitalSalesRooms-checklist.png)
```
Comment (sushmangupta) [inline products/guided-shopping/checklist.md:22]: This needs to be changed

File: products/guided-shopping/installation.md [hunk 3]
```diff
@@ -39,15 +43,15 @@ You can install the plugin via the admin panel or terminal server.
...
 
 1. Click on the “Upload extension” button and choose the zip file containing the plugin from your device.
 
-   ![ ](../../.gitbook/assets/products-guidedShopping-uploadExtension.png)
+   ![ ](../../.gitbook/assets/products-digitalSalesRooms-uploadExtension.png)
 
 1. Once it is uploaded and listed, click “Install”.
 
-    ![ ](../../.gitbook/assets/products-guidedShopping-swagExtensionInstall.png)
+    ![ ](../../.gitbook/assets/products-digitalSalesRooms-swagExtensionInstall.png)
```
Comment (sushmangupta) [inline products/guided-shopping/installation.md:54]: This images needs to be changed

File: products/guided-shopping/installation.md [hunk 4]
```diff
@@ -89,7 +93,7 @@ Detailed below is the minimum configuration needed for a working stack apart fro
 
 * **Other settings** - Take a look at the below sample Mercure configuration on StackHero (Default settings is recommended).
 
-![ ](../../.gitbook/assets/products-guidedShopping-mercureConfig.png)
+![ ](../../.gitbook/assets/products-digitalSalesRooms-mercureConfig.png)
```
Comment (sushmangupta) [inline products/guided-shopping/installation.md:96]: This image also needs to be changed

File: products/guided-shopping/sampleMercureConfig.md [hunk 5]
```diff
@@ -1,3 +1,3 @@
 # Sample Mercure Configuration on StackHero
 
-![Mercure configuration](../../.gitbook/assets/products-guidedShopping-mercureConfig.png)
+![Mercure configuration](../../.gitbook/assets/products-digitalSalesRooms-mercureConfig.png)
```
Comment (sushmangupta) [inline products/guided-shopping/sampleMercureConfig.md:3]: This image needs to be changed

File: products/guided-shopping/installation.md [hunk 6]
```diff
@@ -1,16 +1,20 @@
...
 
-To install and use the Guided Shopping feature, follow the below steps:
+To install and use the DSR feature, follow the below steps:
 
 ### Get the plugin
 
-1. Clone or download the [guided-shopping repository](https://gitlab.com/shopware/shopware/shopware-6/services/swagguidedshopping).
+{% hint style="warning" %}
+The term "Guided Shopping" has been rebranded as "Digital Sales Rooms" (DSR), though no modifications have been implemented in the source code at this point.
+{% endhint %}
```
Comment (sushmangupta) [inline products/guided-shopping/installation.md:15]: Added a hint here, hope this is fine. :

File: products/guided-shopping/README.md (hunk 1 above)
Comment (rittou) [reply products/guided-shopping/README.md:17]: It could be Symfony Flex, I think.

**Review by Isengo1989 — APPROVED** [review]
Thank you for the PR :)

**Pull request merged**